.github/workflows/test-all-warehouses.yml

```diff
@@ -37,29 +37,28 @@ jobs:
       matrix:
         dbt-version:
           ${{ inputs.dbt-version && fromJSON(format('["{0}"]', inputs.dbt-version)) ||
-          ! contains(github.event_name, 'pull_request') && fromJSON('["1.3.0", "latest_official"]') ||
+          ! contains(github.event_name, 'pull_request') && fromJSON('["1.7.0", "latest_official"]') ||
           fromJSON('["latest_official"]') }}
         warehouse-type:
           [
             postgres,
             snowflake,
             bigquery,
             redshift,
-            databricks,
             databricks_catalog,
             athena,
             trino,
           ]
         include:
-          # If we're not running on a specific dbt version, then always add postgres on 1.3.0
-          - dbt-version: "${{ inputs.dbt-version || '1.3.0' }}"
+          # If we're not running on a specific dbt version, then always add postgres on 1.7.0
+          - dbt-version: "${{ inputs.dbt-version || '1.7.0' }}"
             warehouse-type: postgres
           - dbt-version: "${{ inputs.dbt-version || 'latest_pre' }}"
             warehouse-type: postgres
         exclude:
-          - dbt-version: "1.3.0"
+          - dbt-version: "1.7.0"
             warehouse-type: athena
-          - dbt-version: "1.3.0"
+          - dbt-version: "1.7.0"
```

Comment:
wasnt the reason that they were excluded that the adapters did not exist in `dbt 1.3`? if so doesnt it make sense to remove the exclusion?